js/server_functions.js

```diff
@@ -0,0 +1,127 @@
+const fetch = require("./fetch");
+const fs = require("fs");
+const path = require("path");
+const Log = require("logger");
+
+/**
+ * Gets the config.
+ *
+ * @param {Request} req - the request
+ * @param {Response} res - the result
+ */
+function getConfig(req, res) {
+	res.send(config);
+}
+
+/**
+ * A method that forewards HTTP Get-methods to the internet to avoid CORS-errors.
+ *
+ * Example input request url: /cors?sendheaders=header1:value1,header2:value2&expectedheaders=header1,header2&url=http://www.test.com/path?param1=value1
+ *
+ * Only the url-param of the input request url is required.
+ *
+ * @param {Request} req - the request
+ * @param {Response} res - the result
+ */
+async function cors(req, res) {
+	try {
+		const urlRegEx = "url=(.+?)$";
+		let url = "";
+
+		let match = new RegExp(urlRegEx, "g").exec(req.url);
+		if (!match) {
+			url = "invalid url: " + req.url;
+			Log.error(url);
+			res.send(url);
+		} else {
+			url = match[1];
+
+			const headersToSend = GetHeadersToSend(req.url);
+			const expectedRecievedHeaders = GeExpectedRecievedHeaders(req.url);
+
+			Log.log("cors url: " + url);
```

Comment:
Do we really need to log this? Or just make it a debug maybe?

Reply:
I think this was there before and yes this should be logged for supporting stuff, its helpful as hint for supporters when reviewing log files of others e.g. in the forum.